### PR TITLE
xen: use xen-v4.12 recipe as a separate one

### PR DESCRIPTION
**Pull request by iartemenko**
Newer revision of meta-virtualization layer already contains
recipe for xen-v4.10 so we have got conflict duplication
xen-libxentoolcore in PACKAGES variable since
commit: 8cf0a585e04482627c22a70585f64b183725c370 so,
we have to split them up.

Signed-off-by: Iurii Artemenko <iurii_artemenko@epam.com>

### CODE REVIEW

**Comment by andr2000** [comment]
we still have lots of duplication here, e.g. 4.12 has a copy of 4.10, but the offending xen-libxentoolcore.
I see two ways to resolve this:
1. Assume 4.10 is not gonna be used anymore and consider it unsupported. This way duplication could be ok and will only be maintained in 4.12 inc file
2. Move all still common code into version independent inc file, e.g. xen-rocko.inc or so

**Comment by iartemenko** [comment]
Done

File: recipes-domx/meta-xt-images-domx/recipes-extended/xen/xen-4.10-rocko.inc [hunk 1]
```diff
@@ -1,71 +1,10 @@
-FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
-
+require xen-rocko.inc
```
Comment (andr2000) [inline recipes-domx/meta-xt-images-domx/recipes-extended/xen/xen-4.10-rocko.inc]: this wants to be xen-common.inc, not rocko

Comment (andr2000) [reply recipes-domx/meta-xt-images-domx/recipes-extended/xen/xen-4.10-rocko.inc]: with this fixed:
`Reviewed-by: Oleksandr Andrushchenko <oleksandr_andrushchenko@epam.com>`

**Pull request merged**